### DISCUSSION TITLE
prevabs operation issue

### DISCUSSION THREAD

**Discussion by vacupenda** [wenbinyugroup/.github#33] [VABS Related Discussions] [answered]
I can run a file on my own computer using the command 'prevabs -i main_cs_set_0.xml -h -v'. However, when I transfer the file to another computer, it doesn't work—it gets stuck at the third image. On the other hand, the examples from the prevabs installation package (the first image) run without any issues on that computer. I’m not sure how to resolve this problem.
![p3](https://github.com/user-attachments/assets/8f3bffa2-7c17-4a22-88c7-c58206ff2b4c)
![P1](https://github.com/user-attachments/assets/088f26f9-b170-46f2-a692-ca2987820e99)
![P2](https://github.com/user-attachments/assets/b52e2f93-90b5-460f-ac25-8e2ac5027f8a)

**Comment by unitasium** [comment, marked as answer]
If the issue has not been resolved yet, you can try to use the same version of prevabs on both machines. There is also a new version (1.6.0) released recently that you can give it a try. It mainly resolved a vabs dll issue. The older version of prevabs will throw a dll linkage error if you are using a newer version of vabs.

**Comment by vacupenda** [comment]
Thank you ,when I use the same version, it is able to run.
